src/bin/club-install

```diff
@@ -12,16 +12,27 @@ program
     .parse(process.argv);
 
 if (!extant || program.force) {
-    log('API Token (https://app.clubhouse.io/tubi/settings/account/api-tokens): ');
+    const schema = {
+        properties: {
+            mentionName: {
+                message: 'Mention Name -> https://app.clubhouse.io/xxxx/settings/account/ : ',
+                required: true
+            },
+            token: {
+                message: 'API Token -> https://app.clubhouse.io/xxxx/settings/account/api-tokens : ',
```

Comment:
the ` : ` endings here are unnecessary, as they are already appended by `prompt`:

```sh
$ ./src/bin/club install --force
clubhouse: Mention Name -> https://app.clubhouse.io/xxxx/settings/account/ : :  
```